src/pages/error/index.tsx

```diff
@@ -0,0 +1,40 @@
+import DashboardIcon from '@mui/icons-material/Dashboard';
+import Button from '@mui/material/Button';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import ErrorGif from 'assets/cat-gato.gif';
+import Routes from 'routes';
+
+const ErrorPage: React.FC = () => {
+  return (
+    <>
+      <div>
+        <h1>Oops! You seem to be lost.</h1>
+        <Gif />
+        <DashboardButton />
+      </div>
+    </>
```

Comment:
Nie musimy mieć <></>

```suggestion
      <Container>
        <h1>Oops! You seem to be lost.</h1>
        <Gif />
        <DashboardButton />
      </Container>
      
      
const Container = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  gap: 1rem;
`;
```